### DISCUSSION TITLE
How do I add nocalhost cluster？

### DISCUSSION THREAD

**Discussion by yanxiao71** [coder/code-server#5571] [Q&A]
After I clicked the “+” button, the corresponding add screen did not pop up。The following figure：
![image](https://user-images.githubusercontent.com/43506525/190968706-b178a1e4-db67-4c62-8632-ee4684f634a6.png)
![image](https://user-images.githubusercontent.com/43506525/190968829-e4082fb9-d16f-4736-97bf-d292c0d1bf76.png)

How can I solve this problem？

**Comment by jsjoeio** [comment]
Weird. Are there any logs in your browser console? Are you using HTTPS with your code-server deployment?

**Reply by yanxiao71** [reply]
Not serving HTTPS

**Reply by yanxiao71** [reply]
![image](https://user-images.githubusercontent.com/43506525/191150097-def5fa52-0747-4285-82a1-aeb0f6329727.png)

logs

**Reply by yanxiao71** [reply]
[root@mgtdevops02v code-server-4.7.0-linux-amd64]# cat ~/.config/code-server/config.yaml
bind-addr: 0.0.0.0:8081
auth: password
password: root1234
cert: false

**Reply by yanxiao71** [reply]
![image](https://user-images.githubusercontent.com/43506525/191155017-b2696c5e-3043-4049-968e-97f3623ed014.png)

**Reply by yanxiao71** [reply]
The error I found when I clicked the button “+”

**Reply by jsjoeio** [reply]
What happens if you serve with HTTPS (you could use ngrok or localtunnel) and try again? I think it might work. Similar error messages here: https://github.com/coder/code-server/issues/5399
